compiler/aarch64/codegen/GenerateInstructions.hpp

```diff
@@ -1187,6 +1187,26 @@ TR::Instruction *generateVectorShiftImmediateInstruction(
                   uint32_t shiftAmount,
                   TR::Instruction *preced = NULL);
 
+/**
+ * @brief Generates vector unsigned extend long instruction
+ *
+ * @param[in] cg          : CodeGenerator
+ * @param[in] op          : opcode
```

Comment:
This function does not take `op` as an argument.
On the other hand, `node` is missing in this comment.

Reply:
Fixed it.